Lottery-Game-Chainlink-VRF.md

```diff
@@ -11,13 +11,13 @@ So we can use some web2 technologies to generate the randomness and then use the
 ## What is an oracle?
 
 - An oracle sends data from the outside world to a blockchain's smart contract and vice-versa.
-- Smart contract can then use this data to make a decision and change its state.
+- Smart contract can then use this data to make a decision and change it's state.
 - They act as bridges between blockchains and the external world.
-- However it is important to note that the blockchain oracle is not itself the data source but its job is to query, verify and authenticate the outside data and then further pass it to the smart contract.
+- However it is important to note that the blockchain oracle is not itself the data source but it's job is to query, verify and authenticate the outside data and then further pass it to the smart contract.
```

Comment:
This should be `its`. You can find more [here](https://www.grammarly.com/blog/its-vs-its/).